UI/css/nav.css

```diff
@@ -0,0 +1,98 @@
+#navigation{
+    display:flex;
+    justify-content: space-between;
+    color:white;
+}
+
+#nav-container{  /*for other pages except homepage*/
+    padding: 20px 50px 10px;
+    background-color: navy;
+}
+
+nav ul li{
+    list-style: none;
+    display: inline-block;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 600;
+    color:white;
+    margin: 0 10px;
+
+}
+
+nav ul li a{
+    text-decoration: none;
+    color:inherit;
+}
+
+.logo{
+    display: flex;
+}
+
+.menu{
+    display:flex;
+    position: relative;
+}
+.join-btn{
+    background-color: #18227e;
+    border-radius: 30px;
+    padding: 7px 20px;
+}
+.burger-icon{
+    display: none;
+    font-size: 16px;
+}
+
```

Comment:
@Mucyo-Aime-Christian well done, though please never leave unnecessary space in your codebase.